sda-commons-server-testing/src/main/java/org/sdase/commons/server/testing/SystemPropertyRule.java

```diff
@@ -0,0 +1,61 @@
+package org.sdase.commons.server.testing;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+
+/**
+ * This {@link TestRule} allows to set system properties for JUnit tests. All system properties will
+ * be reset after the test has run.
+ *
+ * <p>Example:
+ *
+ * <pre>
+ *   class MyTest {
+ *     &#64;ClassRule public static final SystemPropertyRule PROP = new SystemPropertyRule().setProperty("DISABLE_JWT", "true");
+ *   }
+ * </pre>
+ */
+public class SystemPropertyRule implements TestRule {
```

Comment:
NIT: System Properties can be set in tests using [System Rules](https://stefanbirkner.github.io/system-rules/). We already use them [in some services](https://github.com/search?q=org%3ASDA-SE+stefanbirkner&type=Code) but not here yet. They may also replace our [`EnvironmentRule`](https://github.com/SDA-SE/sda-dropwizard-commons/blob/66e7d99a2bc69e4a39197fa2746f78580677dc15/sda-commons-server-testing/src/main/java/org/sdase/commons/server/testing/EnvironmentRule.java) in the long term.

Reply:
They also provide an `EnvironmentRule`, is it time do deprecate our rule and use this one instead?

Reply:
Nevermind :wink

Reply:
As I said, we should do that in the long term. But that can be a separate PR and should not block this one in my opinion.

Reply:
On the other hand it looks like this would tie us to junit4, so we should probably not deprecated it and stay with our own implementation